modules/ROOT/pages/getting-started-configuring.adoc

```diff
@@ -75,12 +83,32 @@ For an explanation of these terms, see link:https://www.keycloak.org/documentati
 
 :service-name: Identity Management
 
-[[choose-schema]]
-=== Choose the schema of a redirect url
+=== Configuring {idm-name}
+
+. Log into the {mobile-developer-console} and navigate to the {mobile-client} screen.
+
+. Select the *Mobile Services* tab.
+
+. If a binding to the {keycloak-service} service is in progress, a spinning icon is displayed to the right of the {keycloak-service} entry. Wait for the binding process to complete.
+
+. If the _{idm-name} Realm URL_ URL is not visible, expand the Identity Management Service by clicking the > icon.
+
+. Click on the *{idm-name} Realm URL* link to open the {idm-name} Administration Console.
++
+// TODO: @wei will default credentials always be as follows?
+. Log in to the Administration console using the credentials you specified when xref:provisioning[Provisioning] the service (defaults to admin/admin).
+
+. Select `Clients` from the left navigation menu.
++
+// TODO: with a cordova app, I didn't see mobile development platform
+. Select your client from the list of clients. The name of your client is derived from the name of the {mobile-client}, the name of the mobile development platform and the client type, for example `myapp-android-public`.
 
 
+. Add an additional entry to *Valid Redirect URIs*.
```

Comment:
Is it an "input text" filed or what is it that you add the additional entry too?

Reply:
The `Valid Redirect URIs` supports multiple "input text" fields to be added to it.

Reply:
![Snímek obrazovky 2019-06-17 v 10 18 40](https://user-images.githubusercontent.com/13137961/59589075-66eb7f00-90e9-11e9-87c4-8b3edfcd0109.png)

Reply:
Proposal made here https://github.com/aerogear/mobile-docs/pull/707#discussion_r294193666